tax-service-core.yaml

```diff
@@ -180,7 +180,7 @@ components:
             tin:
               $ref: '#/components/schemas/TinDetails'
             address:
-              $ref: '#/components/schemas/Address'
+              $ref: 'https://raw.githubusercontent.com/nium-global/nium-n1-beneficiary-service/514673ddb89af3b05be60c35d33da4c52d0b21ea/core.yaml#/components/schemas/Address'
```

Comment:
|   |   |
|---|---|
|`https://raw.githubusercontent.com/nium-global/nium-n1-beneficiary-service/514673ddb89af3b05be60c35d33da4c52d0b21ea/core.yaml`|`https://raw.githubusercontent.com/nium-global/nium-n1-beneficiary-service/main/core.yaml`|

Your commit using the commit hash of that file vs. the HEAD of the file